src/components/Reports/sharedComponents/ReportPage/components/ReportBlock/index.js

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export, import/named
```

Comment:
Please fix this issue instead of disabling it.

Reply:
fixed